bundles/org.openhab.binding.weathercompany/README.md

```diff
@@ -0,0 +1,1040 @@
+Number:Speed WC_PWS_WindSpeed "Wind Speed [%.0f %unit%]" <wind> { channel="weathercompany:weather-observations:observations:pwsObservations#currentWindSpeed" }
+Number:Speed WC_PWS_WindSpeedGust "Wind Speed Gust [%.0f %unit%]" <wind> { channel="weathercompany:weather-observations:observations:pwsObservations#currentWindSpeedGust" }
+String WC_PWS_Country "Country [%s]" <none> { channel="weathercompany:weather-observations:observations:pwsObservations#country" }
+Location WC_PWS_Location "Lat/Lon [%2$s°N, %3$s°W]" <none> { channel="weathercompany:weather-observations:observations:pwsObservations#location" }
+Number:Length WC_PWS_Elevation "Elevation [%.0f %unit%]" <none> { channel="weathercompany:weather-observations:observations:pwsObservations#elevation" }
+String WC_PWS_Neighborhood "Neighborhood [%s]" <none> { channel="weathercompany:weather-observations:observations:pwsObservations#neighborhood" }
+DateTime WC_PWS_ObservationTimeLocal "Observation Time [%1$tA, %1$tm/%1$td/%1$tY %1$tl:%1$tM %1$tp]" <time> { channel="weathercompany:weather-observations:observations:pwsObservations#observationTimeLocal" }
+Number WC_PWS_QcStatus "QC Status [%.0f %unit%]" <none> { channel="weathercompany:weather-observations:observations:pwsObservations#qcStatus" }
+String WC_PWS_SoftwareType "Software Type [%s]" <none> { channel="weathercompany:weather-observations:observations:pwsObservations#softwareType" }
+String WC_PWS_StationId "Station Id [%s]" <none> { channel="weathercompany:weather-observations:observations:pwsObservations#stationId" }
+
+// Day 0 - Today
+String WC_Day0_DayOfWeek "Day of Week [%s]" <time> { channel="weathercompany:weather-forecast:forecast:forecastDay0#dayOfWeek" }
+DateTime WC_Day0_ValidTimeLocal "Valid At [%1$tA, %1$tm/%1$td/%1$tY]" <time> { channel="weathercompany:weather-forecast:forecast:forecastDay0#validTimeLocal" }
+DateTime WC_Day0_ExpirationTimeLocal "Expires At [%1$tA, %1$tm/%1$td/%1$tY %1$tl:%1$tM %1$tp]" <time> { channel="weathercompany:weather-forecast:forecast:forecastDay0#expirationTimeLocal" }
+String WC_Day0_Narrative "Narrative [%s]" <sun_clouds> { channel="weathercompany:weather-forecast:forecast:forecastDay0#narrative" }
+Number:Temperature WC_Day0_TemperatureMin "Low Temperature [%.1f %unit%]" <temperature> { channel="weathercompany:weather-forecast:forecast:forecastDay0#temperatureMin" }
+Number:Temperature WC_Day0_TemperatureMax "High Temperature [%.1f %unit%]" <temperature> { channel="weathercompany:weather-forecast:forecast:forecastDay0#temperatureMax" }
+Number:Length WC_Day0_PrecipitationRain "Forecasted Rainfall Amount [%.2f %unit%]" <rain> { channel="weathercompany:weather-forecast:forecast:forecastDay0#precipitationRain" }
+Number:Length WC_Day0_PrecipitationSnow "Forecasted Snowfall Amount [%.2f %unit%]" <snow> { channel="weathercompany:weather-forecast:forecast:forecastDay0#precipitationSnow" }
+
+// Day 1 - Tomorrow
```

Comment:
In order to keep it readable, I would strongly suggest to really see this as an example and only show a single day in this readme. The user should be clever enough to understand that he can just copy&paste&replace the section for additional days.

Reply:
Agreed. Was trying to make it easy for users to get started, but I understand your point.